pkg/clients/device/device.go

```diff
@@ -50,6 +50,7 @@ type Client interface {
 type PortsClient interface {
 	DeviceToNetworkType(string, string) (*packngo.Device, error)
 	DeviceNetworkType(string) (string, error)
+	ConvertDevice(*packngo.Device, string) error
```

Comment:
There is a weird relationship here: https://github.com/packethost/packngo/pull/229/files#diff-2121eef3cf61b5f3a4c8e545d742307a5f22b20a5952f964ad5c41d1d252676bL323-R322

DeviceToNetworkType just calls ConvertDevice, I could likely remove ConvertDevice here, and the Mocking around it, `UpdatedInstanceNetworkType` is the test that depends on it most, and I don't think that test is using the mocking as intended (yet, still passing).

I'm going to clean this up before removing draft status on the PR.

Reply:
Cleaned up. Mostly.

I left ConvertDevice and MockConvertDevice references in place with https://github.com/packethost/packngo/issues/230 in mind.